.vitepress/constants/constants.js

```diff
@@ -10,7 +10,7 @@ const constants = Object.freeze({
   rollkitIgniteAppVersion: "rollkit/v0.2.1",
 
   localDALatestTag: "v0.3.1",
-
+  localSequencerLatestTag: "v0.4.0",
```

Comment:
This is pulling for `go-sequencer` repo right? So even though in that repo we are launching a local sequencer, I would change this variable name to refer to the repo for clarity. 
Since local da has its own repo.